OpenDreamRuntime/Objects/MetaObjects/DreamMetaObjectMob.cs

```diff
@@ -39,6 +39,7 @@ sealed class DreamMetaObjectMob : IDreamMetaObject {
             ParentType?.OnVariableSet(dreamObject, varName, value, oldValue);
 
             if (varName == "key" || varName == "ckey") {
+                // TODO: make this work with ckeys too
```

Comment:
I can try to implement this in this PR but if that's wanted I'd like some guidance on what's a good place to store and preserve the ckey to key dictionary. My first instinct was to subclass the player manager but it's sealed and also idk how it all exactly works.

Reply:
It may be better to just loop through DreamManager.Connections and ckey() each username to compare with. I don't think performance would be a huge concern here.